src/components/MenuBar/ButtonBar.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+
+import LinkNoStyle from '../LinkNoStyle/LinkNoStyle';
+
+const styles = theme => ({
+    btn: {
+        fontFamily: theme.typography.fontFamily,
+        fontWeight: 500,
+    },
+});
+
+function ButtonBar(props) {
+    const { classes, isMobile } = props;
```

Comment:
```suggestion
function ButtonBar({ classes, isMobile }) {
```

This gets rid of the `props` variable that we shadowed later in the Link components.